react/OrderForm.tsx

```diff
@@ -77,11 +80,17 @@ const getLocalOrderForm = (): OrderForm | null => {
 
 export const OrderFormProvider: FC = ({ children }) => {
   const { logSplunk } = useSplunk()
+  const runtime = useRuntime
+  const page = runtime().page
```

Comment:
this doesn't seem right, you can use the hook directly without reassigning it to another variable
```suggestion
  const { page } = useRuntime()
```

Reply:
I tried doing this but I get the following message
> Property 'page' does not exist on type '() => PublicRuntime'.

Reply:
Did you run `vtex setup --typings`?

Reply:
Btw I think some things will crash after doing this :/, but check if it solves at least for this part of code, pls

Reply:
it still doesn't work, I think its because you need to do `runtime().page` instead of `runtime.page`

Reply:
I didn't see the type error you commented above @silviasfon, can you post a screenshot?

Reply:
It's fixed hahaha

Reply:
nice!